src/api/routes/tasks.ts

```diff
@@ -17,4 +17,24 @@ router.get("/", (req: Request, res: Response, next: NextFunction) => {
   }
 });
 
+//GET request /tasks/index
+router.get("/:index", (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const idx = req.params.index;
+    console.log("index: ", idx)
+    console.log("data: ", data.data.length)
+    if (data.data.length > idx) {
+      res.status(200).json(data.data[idx]);
```

Comment:
That's not it, @Samridhi-98 ! As I mentioned previously, the response format should be consistent. The task object must be returned as the `data` attribute of an object.
```js
{
	"data": data.data[idx]
}
```
Hence, what you should be doing is:
```js
res.status(200).json({ data: data.data[idx] });
```
Whether an array (for `/` or `?count`) or a single value `/:index`, you need to make sure the response will be an object with the data as its `data` property in JSON.

Reply:
Okay, will update it.